sw/host/opentitantool/src/command/hello.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use erased_serde::Serialize;
+use structopt::StructOpt;
+
+use crate::command::CommandDispatch;
+use opentitanlib::transport::Transport;
+
+/// Sample command heirarchy.
```

Comment:
I think you meant these to be file comments with `//!`?

Reply:
Yes - That block should apply to the module, where as the rest of the comments should be attached to their particular structs.
Done.